catroid/src/org/catrobat/catroid/utils/Utils.java

```diff
@@ -680,4 +680,8 @@ public static boolean isUserLoggedIn(Context context) {
 					|| token.equals(ServerCalls.TOKEN_CODE_INVALID));
 		return tokenValid;
 	}
+
+	public static String getNumberStringForBricks(float value) {
+		return (int) value == value ? "" + (int) value : "" + value;
```

Comment:
This method returns the display string for brick values. It gets rid of the comma and everything behind, if the float value is an integer. This is a requirement by the UX team for this issue

e.g. 15.0f > "15"
4.00000000f > "4"
4.31f > "4.31"